src/styles/_header.scss

```diff
@@ -0,0 +1,73 @@
+body,
+ul,
+ol,
+li {
+  font-size: 12px;
```

Comment:
Setting the font size to 12px globally might affect readability. Consider using relative units like 'rem' or 'em' for better scalability and accessibility.